### ISSUE TITLE
Ember table fluid column reorder incompatibility with Ember 1.8.1

### ISSUE THREAD

**Issue by mariuserik** [Addepar/ember-table#243]
Hi everyone.

After I've updated my application to ember 1.8.1, I got many problems when I reorder columns.

Always when I move a column to the first or to the last position, all my other columns disappear, and my table get messy.

I get the official example in jsbin from Addepar github page

and just changed this line:
``` <script src="http://builds.emberjs.com/tags/v1.4.0/ember.js"></script> ```
to this:
``` <script src="http://builds.emberjs.com/tags/v1.8.1/ember.js"></script> ```

And I get the same problem as in my application, as in screenshot:

![ember-table-problem](https://cloud.githubusercontent.com/assets/1163058/5613520/ae01496a-94e5-11e4-9f37-e2619d24cbb7.png)

I always get this problem with "previousSibling" or "nextSibling" depends if I move a column to the first place or the last 

This is the updated and problematic jsbin example:
http://emberjs.jsbin.com/misatufafe/1/edit?output

anyone have any solution/workaround to this problem?

Best Regards,

Marius

**Comment by azirbel** [comment]
Hey, thanks so much for posting this! I don't know of a workaround , but I'll plan to take a look at it within the next couple days. I recently made some changes to column reordering, but it looks like the table breaks both with and without those changes.

**Comment by mariuserik** [comment]
Hi @azirbel.

I appreciate your answer. And whatever i find something else on this topic i'll keep you updated.

**Comment by mariuserik** [comment]
I've fixed this issue and sent a PR. 

https://github.com/Addepar/ember-table/pull/262/commits

**Comment by azirbel** [comment]
Fixed by https://github.com/Addepar/ember-table/pull/265.

**Comment by mariuserik** [comment]
@azirbel I've just removed my fix and tested the master branch, the reordering seems to work but I get some errors:
![reorder_columns](https://cloud.githubusercontent.com/assets/1163058/5899454/92aa384e-a55b-11e4-8cb9-7ea872201ac4.png)

Should I just ignore them?

**Comment by azirbel** [comment]
Thanks for reporting, I'm seeing the problem too. I'm able to reproduce it by reordering the last column when the table is in fluid mode. I think this is happening because of some bug that miscalculates which columns are resizable. In fluid mode, the last column is not resizable because there's no column to the right it can steal width from. When you reorder it, it looks like the column is still not resizable, which is a bug and seems to be causing this error as well.

**Comment by azirbel** [comment]
This seems related: https://github.com/Addepar/ember-table/pull/266 but I don't believe it solves the underlying problem. Taking a look now.

**Comment by azirbel** [comment]
Confirming that #266 removes the errors, but doesn't fix the problem (after moving the last column, it is still non-resizable).

**Comment by azirbel** [comment]
@mariuserik I just pushed a fix - https://github.com/Addepar/ember-table/pull/268. Can you see if it solves the problem for you?

**Comment by mariuserik** [comment]
Hi @azirbel.

In the first moment everything was appearing well.

But in my project I've the option to regenerate the same table with new data.

So... when I clicked to generate the table for the second time (with new data) I got a error from destroy method again.

![destroy _problem](https://cloud.githubusercontent.com/assets/1163058/5916724/82e0743c-a614-11e4-84e4-5c5f5e506ec4.png)

Any clue?

BR,

Marius Edelsbrunner
